hibernate-core/hibernate-core.gradle

```diff
@@ -118,10 +118,6 @@ jar {
             // TODO: Shouldn't have to explicitly list this, but the plugin
             // generates it with a [1.0,2) version.
             "javax.persistence;version=\"${project.jpaVersion.osgiName}\"",
-            // Temporarily support JTA 1.1 -- Karaf and other frameworks still
-            // use it.  Without this, the plugin generates [1.2,2).
-            // build.gradle adds javax.transaction for all modules
-            'javax.transaction.xa;version="[1.1,2)"',
```

Comment:
So if we remove that, the dependency is added automatically?

Reply:
Yes it is added with version interval: [1.2,2)